Herc mods

illzbee opened the pull request:
<!-- place 'x' mark between square [] brackets to affirm: -->
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

illzbee committed:
added Herc armor set into item_mods.sql

added Herc set
item IDs
helm 25642
body 25718
legs 25842
gloves 27140
boots 27496

illzbee committed:
added herc set mods

added herc set modifiers to armor

cocosolos commented on sql/item_mods.sql:
```diff
@@ -34401,6 +34436,23 @@ INSERT INTO `item_mods` VALUES (25823,31,48);     -- Magic Evasion+48
+INSERT INTO `item_mods` VALUES (25842,384,600);   -- Haste+6%
+INSERT INTO `item_mods` VALUES (25842,73,4);      -- Store TP+4
+INSERT INTO `item_mods` VALUES (25842,27,-4);     -- Enmity-4
+INSERT INTO `item_mods` VALUES (25860,161,-2);    -- Physical Damage Taken-2%
```
Looks like wrong ID slipped in here.

illzbee replied:
oops uploaded the same file twice... the second was supposed to be that fix -_-

illzbee committed:
Add files via upload

illzbee committed:
Delete item_mods.sql

illzbee committed:
Delete item_mods.sql

illzbee committed:
Added herc armor mods

added Herc set mods
item 	IDs 
helm 	25642
body 	25718
legs   	25842
gloves	27140
boots	27496
